projects/kalidea/kaligraphi/src/lib/atoms/kal-option/kal-option.component.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  ViewEncapsulation
+} from '@angular/core';
+import { Highlightable } from '@angular/cdk/a11y';
+
+@Component({
+  selector: 'kal-option',
+  templateUrl: './kal-option.component.html',
+  styleUrls: ['./kal-option.component.sass'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class KalOptionComponent implements OnInit, Highlightable {
+
+  /**
+   * The unique ID of the option
+   */
+  @Input() id?: any;
```

Comment:
I think that you don't need this, have a look at the `uniq ` utils function if it can fits to your need.